src/dex-ui/services/DexService/dao/DAOService.ts

```diff
@@ -415,6 +415,12 @@ export async function fetchMultiSigDAOLogs(daoAccountId: string): Promise<ethers
           ["address", "address tokenAddress"],
           ethers.utils.hexDataSlice(event.args.info.data, 4)
         );
+      } else if (transactionType === MultiSigProposeTransactionType.HBARTokenTransfer) {
+        parsedData = {
+          amount: event.args.info.value,
+          receiver: event.args.info.to,
+          token: TokenId.fromString(HBARTokenId).toSolidityAddress(),
```

Comment:
Wouldn't this send "wrapped" HBAR tokens instead of native hbar? hbar does not have a token ID (as far as I'm aware).

Reply:
yeah you are right SC handles this automatically from txn hash of this `proposal` mean they get all the information to send Native HBARS, this code is just to show the transaction details in UI only this would not effect the actual transaction of sending native hbars from safe to receiver.

Reply:
UI code before making SC calls checks if HBAR tokens get transferred, then invoke different SC mentioned. For non-Hbar tokens, UI invokes the different function. This decision is taken at service layer.

@damiano-melcarne-lab49  Hope this answers your question, we need to merge this any further comment we can address in separate PR.